autoscaler/README.md

```diff
@@ -0,0 +1,37 @@
+# Horizontal Self Scaler container
+
+This container image watches over the number of schedulable nodes in the cluster and resizes
+the number of replicas in the required controller.
+
+Usage of pod_nanny:
+    --configmap <params>
+    --rc <replication-controller>
```

Comment:
You shouldn't need 3 flags.  we should be able to take a single string like "replicationcontrollers/foobar" or "deployments/whatever" and find the scale subresource.  kubectl makes it work...